app/controllers/flipflop/strategies_controller_decorator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# OVERRIDE Flipflop v2.7.1 to allow for custom `Action` labels
+
+module Flipflop
+  module StrategiesControllerDecorator
```

Comment:
I suspect that this might be something to push back to Hyku Prime.   I believe @kirkkwang worked on this regarding how we say PDF.js or IIIF Print splitting.